recipes/vegafusion-jupyter/meta.yaml

```diff
@@ -0,0 +1,47 @@
+{% set name = "vegafusion-jupyter" %}
+{% set version = "0.0.4.post1" %}
```

Comment:
That's a slightly unusual version number! I'm guessing there was a packaging issue with the `0.0.4` pypi release and you didn't want to bump the versions of all the packages in the mono-repo.

Since it aligns with what's released on [pypi](https://pypi.org/project/vegafusion-jupyter/#history) I think this is fine.

Reply:
Just as an aside, in other mono-repos where they publish multiple packages (e.g. [`azure-sdk-for-python`](https://github.com/Azure/azure-sdk-for-python)) I've seen them use prefix-namespaced tags - i.e. `<package-name>_1.2.3` which allows them to version (and release) the subpackages independently. Just mentioning in case it's useful.